### ISSUE TITLE
docker-free installation seems to be light on documentation

### ISSUE THREAD

**Issue by bby-bishopclark** [openspeedtest/Speed-Test#70]
Hi!

I was looking at this to set up trivially internally, and aside from the (fixable) issue of no immutable artifacts to install (ie packages) I see no simple instructions.

- [ ] package build - me

Aside from taking apart the Alpine docker config and emulating that on Enterprise Linux (even photon), what other instructions do we have in some consolidated and hyper-legible format?

 - [ ] installation idiodirections (lather, rinse, repeat) - needed

I'll probably find some the moment I submit this ticket, so watch for deletion.  With luck I'll win budget this week for the installable bit.

**Comment by openspeedtest** [comment]
1) All you need to do is use Nginx configuration I posted.
2) Download this project and extract it to www dir. 

Just like how you run a static Webpage.

**Comment by bby-bishopclark** [comment]
I saw the reference to the nginx config in the docker project, for sure.

For #2 , files sprayed onto a filesystem with no validation options was deemed risky a while ago - even for files sprayed in as lowly static files on day one - because we want to be able to confirm they're still just static files like we deployed on day one (with signed checksums).  Atomic install, explicit dependencies, clean removal, other features, blah blah.

No worries.  The packaging is simple, and you don't have to accept it back if it's out of scope; all good.  

Thanks for the 1-2 install hints!
